tests/behaviour/steps/cli.py

```diff
@@ -17,7 +17,6 @@ def step_impl(context, arguments: str):
     dirs = AppDirs("csvcubed-cli", "csvcubed")
     context.csvcubed_log_location = Path(dirs.user_log_dir)
 
-
```

Comment:
Hardly the worst problem, but the black formatter should have insisted you keep this line. It likes 2 lines between functions.

Reply:
You are correct - I had to reset my black formatter and the formatting works as expected now.